scipy/special/functions.json

```diff
@@ -1402,6 +1402,21 @@
             "cdft2_wrap": "dd->d"
         }
     },
+    "_stirling2_inexact": {
+        "stirling2.h++": {
+            "_stirling2_inexact": "dd->d"
+        }
+    },
+    "_stirling2_temme": {
+        "stirling2.h++": {
+            "_stirling2_temme": "dd->d"
+        }
+    },
+    "_stirling2_dp": {
+        "stirling2.h++": {
+            "_stirling2_dp": "dd->d"
+        }
+    },
```

Comment:
should get rid of these too

Reply:
removed `_stirling2_dp` and `_stirling2_temme` but left the `_stirling2_inexact` because we need that one for the `stirling2` python function to call.